#Request
---
#Response
geometry_msgs/PoseWithCovarianceStamped pose